templates/home/Html/Links/link.html

```diff
@@ -1,14 +1,276 @@
-{% extends 'home/base.html' %}
-{% load static %}
-
-{% block content %}
```

Comment:
Why have you removed these lines? We were using template inheritance.

Reply:
The thing is that I needed a fixed top navbar for the courses' pages for my side navbars to function the best. So, if I would change the navbar code in original inheritance file it would change the navbar style for all other pages as well which I wanted to avoid. So now, the templates that I have created have the fixed topbar & side navbar customised adaptations and we will just have to add that using inheritance in all the courses' webpages, nowhere else.